cartography/data/jobs/cleanup/aws_import_organization_cleanup.json

```diff
@@ -0,0 +1,15 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (o:AWSOrganization) WHERE o.lastupdated <> {UPDATE_TAG} WITH o LIMIT {LIMIT_SIZE} DETACH DELETE (o) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100
+    },
+    {
+      "query": "MATCH (:AWSAccount)-[r:BELONG_TO]->(:AWSOrganization) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DETACH DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
Nit: The convention for resource hierarchy seems to be defined with `:RESOURCE`, so `(:AWSOrganization)-[:RESOURCE]->(:AWSAccount)-[:RESOURCE]->...`.

I admittedly don't have a very strong opinion on this but thought I'd bring it up for consideration.